docs/alerts/monitors/settings.md

```diff
@@ -46,9 +46,9 @@ In the details pane, you'll see the following details for a Monitor:
 * **Type**. Shows the type of Monitor, either Logs or Metrics.
 * **Path**. Shows the Library location where the monitor is located.
 * **Query**. It is used to track your data. 
-* **Trigger Conditions** Thresholds value that must met for Monitor to trigger an alert. These values are set when you create a monitor and can be based on a variety of metrics such as CPU usage, network latency, application response time. . Applicable values include Critical, Warning, and Missing Data.
+* **Trigger Conditions** Thresholds value that must met for Monitor to trigger an alert. These values are set when you create a monitor and can be based on a variety of metrics such as CPU usage, network latency, application response time. Applicable values include Critical, Warning, and Missing Data.
 * **Notifications**. These are configured on the Monitor.
-* The timestamp and user that **Created** and last **Modified** the Monitor.
+* The timestamp and user that **Created** and last **Modified** the Monitor. The alert response page will show alert details in the time zone designated in your user preferences.
```

Comment:
@ndhar-sumo regarding your comment [here](https://github.com/SumoLogic/sumologic-documentation/pull/3291#issuecomment-1796826518), can you review this change?